experimental/docker-stacks-and-bundles.md

```diff
@@ -52,7 +52,7 @@ Options:
 Let's deploy the stack created before:
 
 ```bash
-# docker deploy vossibility-stack
+# docker deploy vossibility-stack  vossibility
```

Comment:
Yes，Here is my debugging process
```bash
➜  ls 
app.js  bundle.dab  docker-compose.yml  vossibilitystack.dab
➜  docker deploy  vossibilitystack.dab
Please specify either a bundle file (with --bundle-file) or a Compose file (with --compose-file).
➜ docker deploy  --bundle-file vossibilitystack.dab
"docker deploy" requires exactly 1 argument(s).
See 'docker deploy --help'.

Usage:  docker deploy [OPTIONS] STACK

Deploy a new stack or update an existing stack
```
but when i specify a name for the docker deploy stack command,it work well

```bash
➜  docker deploy  --bundle-file vossibilitystack.dab voss
Loading bundle from vossibilitystack.dab
Creating network voss_default
Creating service voss_node
```